### PR TITLE
[stable/nginx-ingress] Support initContainers

### PR DESCRIPTION
**Pull request by JanVoracek**
**What this PR does / why we need it**: 
I needed some initialization (create `/var/log/nginx` on a mounted volume) before the controller starts.

### CODE REVIEW

File: stable/nginx-ingress/values.yaml [hunk 1]
```diff
@@ -219,6 +219,13 @@ controller:
   #  - name: copy-portal-skins
   #    emptyDir: {}
 
+  initContainers: {}
```
Comment (unguiculus) [inline stable/nginx-ingress/values.yaml]: Should be an empty list. Please rename it for consistency. 

`extraInitContainers: [] `

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: stable/nginx-ingress/values.yaml (hunk 1 above)
Comment (JanVoracek) [reply stable/nginx-ingress/values.yaml]: OK. Just a question - shouldn't be `extraContainers`, `extraVolumes`, etc. lists as well? 

https://github.com/kubernetes/charts/blob/6b619a5bfd0c888df7623d5bc390b516d4b6947b/stable/nginx-ingress/values.yaml#L188

**Review by unguiculus — APPROVED** [review]

**Pull request merged**